lib/Cpdf.php

```diff
@@ -3614,6 +3811,116 @@ function fillStroke()
         $this->addContent("\nb" . ($this->fillRule === "evenodd" ? "*" : ""));
     }
 
+    /**
+     * add signature
+     *
+     * $fieldSigId = $cpdf->addFormField('Sig', 'Signature1', 0, 0, 0, 0, 0);
+     *
+     * $signatureId = $cpdf->addSignature([
+     *   'signcert' => file_get_contents('dompdf.crt'),
+     *   'privkey' => file_get_contents('dompdf.key'),
+     *   'password' => 'password',
+     *   'name' => 'DomPDF DEMO',
+     *   'location' => 'Home',
+     *   'reason' => 'First Form',
+     *   'contactinfo' => 'info'
+     * ]);
+     * $cpdf->setFormFieldValue($fieldSigId, "$signatureId 0 R");
+     * $cpdf->addForm([$fieldSigId], 3);
+     *
+     * @param array $certInfo (signcert, privkey, password, name, location, reason, contactinfo)
+     * @return int
+     */
+    function addSignature($certInfo = [])
```

Comment:
Better add all parameters here for readability. Optional ones (if any) could have a default null value.